workflow-templates/docker-ci.yaml

```
@@ -0,0 +1,64 @@
+name: DockerCI
+
+on:
+  pull_request: ~
+  schedule:
+    - cron: '0 8 * * *' # Triggers daily at 800
+
+jobs:
+  build-and-scan-image:
```

```suggestion
  build-and-trivy-scan:
```

In case we add more security-related jobs